docs/tables/azuread_directory_setting.md

```diff
@@ -0,0 +1,47 @@
+# Table: azuread_directory_setting
+
+Directory settings define the configurations that can be used to customize the tenant-wide and object-specific restrictions and allowed behavior.
```

Comment:
Are there any docs from Azure on a list of all possible directory settings?

Reply:
I couldn't locate any reliable docs from Azure listing all of the templates and setting names/values